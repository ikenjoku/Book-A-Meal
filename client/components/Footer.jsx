import React from 'react';
import { ToastContainer } from 'react-toastify';

export default () => (
  <footer className="footer">
    <ToastContainer
      autoClose={15000}
      hideProgressBar={true}
      closeOnClick={true}
      pauseOnHover={false}
      closeButton={false}
      className="toast-container"
    />
    <p>Book-A-Meal &copy; 2018</p>
    <p>Designed by <a href="https://github.com/ikenjoku/Book-A-Meal" target="blank">Ikechukwu Njoku</a></p>
  </footer>
);
