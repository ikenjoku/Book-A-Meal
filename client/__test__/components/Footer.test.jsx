import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';


test('it should render footer correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
