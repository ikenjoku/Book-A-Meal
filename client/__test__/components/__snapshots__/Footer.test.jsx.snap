// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render footer correctly 1`] = `
<footer
  className="footer"
>
  <ToastContainer
    autoClose={4000}
    bodyClassName={null}
    className="toast-container"
    closeButton={false}
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={true}
    newestOnTop={false}
    pauseOnHover={false}
    pauseOnVisibilityChange={true}
    position="top-right"
    progressClassName={null}
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <p>
    Book-A-Meal Â© 2018
  </p>
  <p>
    Designed by 
    <a
      href="https://github.com/ikenjoku/Book-A-Meal"
      target="blank"
    >
      Ikechukwu Njoku
    </a>
  </p>
</footer>
`;
